requirements-dev.txt

```diff
@@ -0,0 +1,21 @@
+appdirs==1.4.3
+atomicwrites==1.3.0
+attrs==19.1.0
+black==19.3b0
+Click==7.0
+entrypoints==0.3
+flake8==3.7.8
+importlib-metadata==0.20
+mccabe==0.6.1
+more-itertools==7.2.0
+packaging==19.1
+pluggy==0.12.0
+py==1.8.0
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pyparsing==2.4.2
+pytest==5.1.2
+six==1.12.0
+toml==0.10.0
+wcwidth==0.1.7
+zipp==0.6.0
```

Comment:
I think `pytest` should be in `requirements-dev.txt`, and CircleCI should use `venvinstall.sh --dev`. Then you wouldn't have to detect `$CI` in `venvinstall.sh`.

Reply:
Cannot be done until Python 3.5 is removed from CircleCI as it throws an error while trying to install black which is in the dev requirements.